### PR TITLE
The Great Ammo Purge

### PR DESCRIPTION
**Pull request by Tron-117**
Removes most ammo types except FMJ.

## About The Pull Request
This is for the eventual gun rework, in which damage is based on the gun itself rather than it's ammo type. The option to make the special ammos have been removed from the ammo bench, and can not even be spawned in via admin means. Some ammo types, like explosive ammo and the ricochet 45 ammo have been kept in, as well as the ammo that's used by raider mobs, which is fine as it's impossible to make those anyway.

## Why It's Good For The Game
Makes balance a heellll of a lot easier.

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
del: Removes near every ammo type except FMJ.
/:cl:

### CODE REVIEW

**Comment by Tron-117** [comment]
Who is the ASS who scattered AP casings about the map aaaaahhhh.

File: code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -193,7 +193,7 @@ GLOBAL_LIST_INIT(trash_ammo, list(
 	/obj/item/ammo_box/magazine/zipgun = 3,
 	/obj/item/ammo_casing/shotgun/buckshot = 2,
 	/obj/item/ammo_box/c38 = 3,
-	/datum/design/ammolathe/a22plink
+	///datum/design/ammolathe/a22plink
```
Comment (Looreydev) [inline code/_globalvars/lists/maintenance_loot.dm:196]:
```suggestion
	
```

File: code/datums/components/storage/concrete/pockets.dm [hunk 2]
```diff
@@ -222,8 +222,9 @@ GLOBAL_LIST_INIT(storage_holster_can_hold, typecacheof(list(
 	/obj/item/ammo_box/a762,
 	/obj/item/ammo_box/a556/stripper,
 	/obj/item/ammo_box/needle,
-	/obj/item/ammo_box/needleap,
+	/*/obj/item/ammo_box/needleap,
 	/obj/item/ammo_box/needleultra,
+	*/
```
Comment (Looreydev) [inline code/datums/components/storage/concrete/pockets.dm:227]:
```suggestion
```

File: code/game/objects/items/storage/belt.dm [hunk 3]
```diff
@@ -721,8 +721,9 @@
 		/obj/item/ammo_box/a762,
 		/obj/item/ammo_box/a556/stripper,
 		/obj/item/ammo_box/needle,
-		/obj/item/ammo_box/needleap,
+		/*/obj/item/ammo_box/needleap,
 		/obj/item/ammo_box/needleultra,
+		*/
```
Comment (Looreydev) [inline code/game/objects/items/storage/belt.dm:726]:
```suggestion
```

File: code/modules/jobs/job_types/wasteland.dm [hunk 4]
```diff
@@ -1222,7 +1222,7 @@ datum/job/wasteland/f13dendoctor
 	suit = /obj/item/clothing/suit/armor/f13/rangercombat/desert
 	r_hand = /obj/item/gun/ballistic/revolver/revolver44/desert_ranger
 	backpack_contents = list(
-							/obj/item/ammo_box/m44box/swc=2
+							/obj/item/ammo_box/m44box=2
```
Comment (Looreydev) [inline code/modules/jobs/job_types/wasteland.dm:1225]:
```suggestion
```

File: code/modules/projectiles/ammunition/ballistic/pistol.dm [hunk 5]
```diff
@@ -5,7 +5,7 @@
 	caliber = "10mm"
 	projectile_type = /obj/item/projectile/bullet/c10mm
 
-/obj/item/ammo_casing/c10mm/ap
+/*/obj/item/ammo_casing/c10mm/ap
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/pistol.dm:8]: Remove this whole commented block too.

File: code/modules/projectiles/ammunition/ballistic/pistol.dm [hunk 6]
```diff
@@ -22,7 +23,7 @@
 	caliber = "9mm"
 	projectile_type = /obj/item/projectile/bullet/c9mm
 
-/obj/item/ammo_casing/c9mm/ap
+/*/obj/item/ammo_casing/c9mm/ap
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/pistol.dm:26]: This too, slash it. (Line 26 to 40)

File: code/modules/projectiles/ammunition/ballistic/pistol.dm [hunk 7]
```diff
@@ -44,10 +46,11 @@
 	caliber = "14"
 	projectile_type = /obj/item/projectile/bullet/mm14
 
-/obj/item/ammo_casing/p14mm/jhp
+/*/obj/item/ammo_casing/p14mm/jhp
 	name = "14mm JHP bullet casing"
 	desc = "A 14mm JHP bullet casing"
 	projectile_type = /obj/item/projectile/bullet/mm14/jhp
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/pistol.dm:53]:
```suggestion
```

File: code/modules/projectiles/ammunition/ballistic/pistol.dm [hunk 8]
```diff
@@ -56,7 +59,7 @@
 	caliber = ".22lr"
 	projectile_type = /obj/item/projectile/bullet/c22
 
-/obj/item/ammo_casing/a22/hp
+/*/obj/item/ammo_casing/a22/hp
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/pistol.dm:62]: Delete this too. (Line 62 to 71)

File: code/modules/projectiles/ammunition/ballistic/revolver.dm [hunk 9]
```diff
@@ -5,13 +5,14 @@
...
 
-/obj/item/ammo_casing/a357/jhp
+/*/obj/item/ammo_casing/a357/jhp
 	name = ".357 JHP bullet casing"
 	desc = "A .357 JHP bullet casing."
 
 /obj/item/ammo_casing/a357/jfp
 	name = ".357 JFP bullet casing"
 	desc = "A .357 JFP bullet casing."
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/revolver.dm:15]:
```suggestion
```

File: code/modules/projectiles/ammunition/ballistic/revolver.dm [hunk 10]
```diff
@@ -27,7 +28,7 @@
 	caliber = "44"
 	projectile_type = /obj/item/projectile/bullet/m44
 
-/obj/item/ammo_casing/m44/jhp
+/*/obj/item/ammo_casing/m44/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/revolver.dm:31]: This block too. (Line 31 to 40)

File: code/modules/projectiles/ammunition/ballistic/revolver.dm [hunk 11]
```diff
@@ -44,7 +46,7 @@
 	caliber = "4570"
 	projectile_type = /obj/item/projectile/bullet/c4570
 
-/obj/item/ammo_casing/c4570/jhp
+/*/obj/item/ammo_casing/c4570/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/revolver.dm:49]: This. (Line 49 to 58)

File: code/modules/projectiles/ammunition/ballistic/rifle.dm [hunk 12]
```diff
@@ -11,7 +11,7 @@
 	desc = "A .308 sporting bullet casing."
 	projectile_type = /obj/item/projectile/bullet/a762/sport
 
-/obj/item/ammo_casing/a762/ap
+/*/obj/item/ammo_casing/a762/ap
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/rifle.dm:14]: This. (Line 14 to 25)

File: code/modules/projectiles/ammunition/ballistic/rifle.dm [hunk 13]
```diff
@@ -30,7 +31,7 @@
 	caliber = "a556"
 	projectile_type = /obj/item/projectile/bullet/a556
 
-/obj/item/ammo_casing/a556/jhp
+/*/obj/item/ammo_casing/a556/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/rifle.dm:34]: This. (Line 34 to 45)

File: code/modules/projectiles/ammunition/ballistic/smg.dm [hunk 14]
```diff
@@ -6,7 +6,7 @@
 	caliber = ".45"
 	projectile_type = /obj/item/projectile/bullet/c45
 
-/obj/item/ammo_casing/c45/jhp
+/*/obj/item/ammo_casing/c45/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/smg.dm:9]: This. (Line 9 to 18)

File: code/modules/projectiles/ammunition/ballistic/sniper.dm [hunk 15]
```diff
@@ -14,11 +14,12 @@
 	caliber = "a50MG"
 	projectile_type = /obj/item/projectile/bullet/a50MG/incendiary
 
-/obj/item/ammo_casing/a50MG/AP
+/*/obj/item/ammo_casing/a50MG/AP
 	name = ".50 MG AP bullet casing"
 	desc = "A .50 MG armor-piercing bullet casing."
 	caliber = "a50MG"
 	icon_state = "50ap2"
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/ballistic/sniper.dm:22]:
```suggestion
```

File: code/modules/projectiles/ammunition/caseless/_caseless.dm [hunk 16]
```diff
@@ -29,7 +29,7 @@
 /obj/item/ammo_casing/caseless/needle/attackby()
 	return
 */
-/obj/item/ammo_casing/caseless/needle/ap
+/*/obj/item/ammo_casing/caseless/needle/ap
```
Comment (Looreydev) [inline code/modules/projectiles/ammunition/caseless/_caseless.dm:32]: This. (Line 32 to 44)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 17]
```diff
@@ -65,7 +65,7 @@
 	w_class = WEIGHT_CLASS_SMALL
 	custom_materials = list(/datum/material/iron = 6000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/m22/plinking
+/*/obj/item/ammo_box/m22/plinking
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:68]: This. (Line 68 to 77)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 18]
```diff
@@ -87,7 +88,7 @@
 	max_ammo = 30
 	custom_materials = list(/datum/material/iron = 15000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/c9mm/ap
+/*/obj/item/ammo_box/c9mm/ap
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:91]: This (Line 91 to 105)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 19]
```diff
@@ -132,20 +134,22 @@
 	max_ammo = 30
 	custom_materials = list(/datum/material/iron = 10000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/c10mm/jhp
+/*/obj/item/ammo_box/c10mm/jhp
 	name = "ammo box (10mm JHP)"
 	ammo_type = /obj/item/ammo_casing/c10mm/jhp
 	custom_materials = list(/datum/material/iron = 6000, /datum/material/blackpowder = 1000)
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:141]:
```suggestion
```

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 20]
```diff
@@ -132,20 +134,22 @@
...
 	name = "ammo box (10mm Incendiary)"
 	icon_state = "10mmbox"
 	max_ammo = 20
 
-/obj/item/ammo_box/c10mm/ap
+/*/obj/item/ammo_box/c10mm/ap
 	name = "ammo box (10mm AP)"
 	ammo_type = /obj/item/ammo_casing/c10mm/ap
 	custom_materials = list(/datum/material/iron = 14000, /datum/material/titanium = 3000, /datum/material/blackpowder = 1000)
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:152]:
```suggestion
```

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 21]
```diff
@@ -167,7 +171,7 @@
 	custom_materials = list(/datum/material/iron = 16000, /datum/material/blackpowder = 1000)
 	w_class = WEIGHT_CLASS_NORMAL
 
-/obj/item/ammo_box/a357box/jhp
+/*/obj/item/ammo_box/a357box/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:174]: This (Line 174 to 183)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 22]
```diff
@@ -190,7 +195,7 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron = 14000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/m44box/jhp
+/*/obj/item/ammo_box/m44box/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:198]: This. (Line 198 to 207)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 23]
```diff
@@ -230,7 +236,7 @@
 	max_ammo = 30
 	custom_materials = list(/datum/material/iron = 10000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/c45/jhp
+/*/obj/item/ammo_box/c45/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:239]: This. (Line 239 to 248)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 24]
```diff
@@ -261,7 +268,7 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron = 16000, /datum/material/blackpowder = 1500)
 
-/obj/item/ammo_box/c4570box/jhp
+/*/obj/item/ammo_box/c4570box/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:271]: This. (Line 271 to 280)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 25]
```diff
@@ -284,7 +292,7 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron = 20000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/a556/jhp
+/*/obj/item/ammo_box/a556/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:295]: This. (Line 295 to 309)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 26]
```diff
@@ -336,7 +345,7 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron = 16000, /datum/material/blackpowder = 1000)
 
-/obj/item/ammo_box/a762box/jhp
+/*/obj/item/ammo_box/a762box/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:348]: This. (Line 348 to 362)

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 27]
```diff
@@ -375,10 +385,11 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron = 11000, /datum/material/blackpowder = 1500)
 
-/obj/item/ammo_box/m14mm/jhp
+/*/obj/item/ammo_box/m14mm/jhp
 	name = "ammo box (14mm JHP)"
 	ammo_type = /obj/item/ammo_casing/p14mm/jhp
 	custom_materials = list(/datum/material/iron = 11000)
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:392]:
```suggestion
```

File: code/modules/projectiles/boxes_magazines/ammo_boxes.dm [hunk 28]
```diff
@@ -680,7 +691,7 @@
 	multiple_sprites = 1
 	custom_materials = list(/datum/material/iron = 3000, /datum/material/glass = 5000)
 
-/obj/item/ammo_box/needleap
+/*/obj/item/ammo_box/needleap
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/ammo_boxes.dm:694]: This. (Line 694 to 711)

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 29]
```diff
@@ -56,11 +56,12 @@
 	max_ammo = 15
 	multiple_sprites = 2
 
-/obj/item/ammo_box/magazine/m9mmds/ap
+/*/obj/item/ammo_box/magazine/m9mmds/ap
 	name = "doublestack AP magazine (9mm)"
 	icon_state = "m9mmds"
 	ammo_type = /obj/item/ammo_casing/c9mm/ap
 	special_ammo = TRUE
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm:64]:
```suggestion
```

File: code/modules/projectiles/boxes_magazines/external/pistol.dm [hunk 30]
```diff
@@ -111,10 +112,11 @@
 	max_ammo = 8
 	multiple_sprites = 1
 
-/obj/item/ammo_box/magazine/m45/ap
+/*/obj/item/ammo_box/magazine/m45/ap
 	name = "premium magazine (.45)"
 	ammo_type = /obj/item/ammo_casing/c45/op
 	special_ammo = TRUE
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/external/pistol.dm:119]:
```suggestion
```

File: code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 31]
```diff
@@ -81,12 +81,13 @@
...
 	max_ammo = 8
 
-/obj/item/ammo_box/magazine/garand308/ap
+/*/obj/item/ammo_box/magazine/garand308/ap
 	name = "en-bloc clip Armor Piercing (7.62x51mm)"
 	icon_state = "enbloc-8"
 	ammo_type = /obj/item/ammo_casing/a762/ap
 	caliber = "a762"
 	max_ammo = 8
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/boxes_magazines/external/rifle.dm:90]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 32]
```diff
@@ -26,7 +26,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	armour_penetration = 0.06
 	wound_bonus = 6
 
-/obj/item/projectile/bullet/c22/hp
+/*/obj/item/projectile/bullet/c22/hp
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:29]: This. (Line 29 to 41)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 33]
```diff
@@ -61,7 +62,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	armour_penetration = 0.05
 	wound_bonus = 15
 
-/obj/item/projectile/bullet/c9mm/ap
+/*/obj/item/projectile/bullet/c9mm/ap
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:65]: This. (Line 65 to 78)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 34]
```diff
@@ -74,13 +75,15 @@ Civilian round				=	-10% damage. AP reduced by 50%
...
 	bare_wound_bonus = 23
+*/
 
 /obj/item/projectile/bullet/c9mm/op
 	name = "9mm +P bullet"
 	damage = 27
 	armour_penetration = 0.07
 	var/extra_speed = 500
 
+
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:86]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 35]
```diff
@@ -95,7 +98,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	armour_penetration = 0.12
 	wound_bonus = 18
 
-/obj/item/projectile/bullet/c10mm/ap
+/*/obj/item/projectile/bullet/c10mm/ap
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:101]: This. (Line 101 to 114)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 36]
```diff
@@ -120,12 +124,13 @@ Civilian round				=	-10% damage. AP reduced by 50%
...
 	wound_bonus = 20
 
-/obj/item/projectile/bullet/c45/jhp
+/*/obj/item/projectile/bullet/c45/jhp
 	name = ".45 JHP bullet"
 	damage = 37
 	armour_penetration = 0
 	wound_bonus = -30
 	bare_wound_bonus = 30
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:133]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 37]
```diff
@@ -145,7 +150,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	wound_bonus = 14
 	bare_wound_bonus = -14
 
-/obj/item/projectile/bullet/a357/jhp
+/*/obj/item/projectile/bullet/a357/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:153]: This. (Line 153 to 166)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 38]
```diff
@@ -171,7 +177,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	wound_bonus = 20
 	bare_wound_bonus = -20
 
-/obj/item/projectile/bullet/m44/jhp
+/*/obj/item/projectile/bullet/m44/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:180]: This. (Line 180 to 193)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 39]
```diff
@@ -201,7 +208,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	wound_bonus = 24
 	bare_wound_bonus = -24
 
-/obj/item/projectile/bullet/c4570/jhp
+/*/obj/item/projectile/bullet/c4570/jhp
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:211]: This. (Line 211 to 224)

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 40]
```diff
@@ -237,12 +245,13 @@ Civilian round				=	-10% damage. AP reduced by 50%
...
 	bare_wound_bonus = -28
 
-/obj/item/projectile/bullet/mm14/jhp
+/*/obj/item/projectile/bullet/mm14/jhp
 	name = "14mm JHP bullet"
 	damage = 52
 	armour_penetration = 0
 	wound_bonus = -42
 	bare_wound_bonus = 42
+*/
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:254]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets/pistol.dm [hunk 41]
```diff
@@ -275,7 +284,7 @@ Civilian round				=	-10% damage. AP reduced by 50%
 	armour_penetration = 0.6
 	var/piercing = FALSE
 
-/obj/item/projectile/bullet/needle/ap
+/*/obj/item/projectile/bullet/needle/ap
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/pistol.dm:287]: This. (Line 287 to 300)

File: code/modules/projectiles/projectile/bullets/rifle.dm [hunk 42]
```diff
@@ -27,7 +27,7 @@ Civilian round				=	-10% damage for .223. AP reduced by 50%
 	wound_bonus = 18
 	bare_wound_bonus = -18
 
-/obj/item/projectile/bullet/a556/ap
+/*/obj/item/projectile/bullet/a556/ap
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/rifle.dm:30]: This. (Line 30 to 43)

File: code/modules/projectiles/projectile/bullets/rifle.dm [hunk 43]
```diff
@@ -74,7 +75,7 @@ Civilian round				=	-10% damage for .223. AP reduced by 50%
 	wound_bonus = 20
 	bare_wound_bonus = -20
 
-/obj/item/projectile/bullet/a762/ap
+/*/obj/item/projectile/bullet/a762/ap
```
Comment (Looreydev) [inline code/modules/projectiles/projectile/bullets/rifle.dm:78]: This. (Line 78 to 97)

File: code/modules/research/designs/ammolathe_designs.dm [hunk 44]
```diff
@@ -141,12 +141,13 @@
...
 
 /datum/design
-/datum/design/ammolathe/a22plink
+/*/datum/design/ammolathe/a22plink
 	name = ".22lr plinking ammo box"
 	id = "a22plink"
 	build_path = /obj/item/ammo_box/m22/plinking
 	materials = list(/datum/material/iron = 3000, /datum/material/blackpowder = 1000)
 	category = list("initial", "Simple Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:150]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 45]
```diff
@@ -155,19 +156,21 @@
...
 	category = list("initial", "Simple Ammo")
 
-/datum/design/ammolathe/a9mmjhp
+/*/datum/design/ammolathe/a9mmjhp
 	name = "9mm JHP ammo box"
 	id = "a9mmjhp"
 	build_path = /obj/item/ammo_box/c9mm/jhp
 	materials = list(/datum/material/iron = 11000, /datum/material/blackpowder = 1000)
 	category = list("initial", "Simple Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:165]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 46]
```diff
@@ -155,19 +156,21 @@
...
 
-/datum/design/ammolathe/a45jhp
+/*/datum/design/ammolathe/a45jhp
 	name = ".45 ACP JHP ammo box"
 	id = "a45jhp"
 	build_path = /obj/item/ammo_box/c45/jhp
 	materials = list(/datum/material/iron = 6000, /datum/material/blackpowder = 1000)
 	category = list("initial", "Simple Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:173]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 47]
```diff
@@ -222,7 +225,7 @@
 	category = list("initial", "Basic Magazines")
 
 //Tier 2 Ammo
-/datum/design/ammolathe/c10mmjhp
+/*/datum/design/ammolathe/c10mmjhp
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:228]: This. (Line 228 to 241)

File: code/modules/research/designs/ammolathe_designs.dm [hunk 48]
```diff
@@ -243,12 +247,13 @@
...
 	category = list("initial", "Basic Ammo")
 
-/datum/design/ammolathe/m44jhp
+/*/datum/design/ammolathe/m44jhp
 	name = ".44 Magnum JHP ammo box"
 	id = "m44jhp"
 	materials = list(/datum/material/iron = 10000, /datum/material/blackpowder = 1500)
 	build_path = /obj/item/ammo_box/m44box/jhp
 	category = list("initial", "Basic Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:256]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 49]
```diff
@@ -257,12 +262,13 @@
...
 	category = list("initial", "Basic Ammo")
 
-/datum/design/ammolathe/a357jhp
+/*/datum/design/ammolathe/a357jhp
 	name = ".357 Magnum JHP ammo box"
 	id = "a357jhp"
 	materials = list(/datum/material/iron = 12000, /datum/material/blackpowder = 1500)
 	build_path = /obj/item/ammo_box/a357box/jhp
 	category = list("initial", "Basic Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:271]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 50]
```diff
@@ -271,12 +277,14 @@
...
 
-/datum/design/ammolathe/a556jhp
+/*/datum/design/ammolathe/a556jhp
 	name = "5.56 JHP ammo box"
 	id = "a556jhp"
 	materials = list(/datum/material/iron = 16000, /datum/material/blackpowder = 1500)
 	build_path = /obj/item/ammo_box/a556/jhp
 	category = list("initial", "Basic Ammo")
+*/
+
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:287]:
```suggestion
```

**Comment by Tron-117** [comment]
Please stop you're flooding my email, I'll delete instead of comment out god damn

File: code/modules/research/designs/ammolathe_designs.dm [hunk 51]
```diff
@@ -292,12 +300,13 @@
...
 	category = list("initial", "Basic Ammo")
 
-/datum/design/ammolathe/a556ap
+/*/datum/design/ammolathe/a556ap
 	name = "5.56 AP ammo box"
 	id = "a556ap"
 	materials = list(/datum/material/iron = 24000, /datum/material/titanium = 6000, /datum/material/blackpowder = 1500)
 	build_path = /obj/item/ammo_box/a556/ap
 	category = list("initial", "Basic Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:309]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 52]
```diff
@@ -306,12 +315,13 @@
...
 	category = list("initial", "Basic Ammo")
 
-/datum/design/ammolathe/a762jhp
+/*/datum/design/ammolathe/a762jhp
 	name = "7.62 JHP ammo box"
 	id = "a762jhp"
 	materials = list(/datum/material/iron = 12000, /datum/material/blackpowder = 1500)
 	build_path = /obj/item/ammo_box/a762box/jhp
 	category = list("initial", "Basic Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:324]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 53]
```diff
@@ -327,12 +337,13 @@
...
 	category = list("initial", "Basic Ammo")
 
-/datum/design/ammolathe/a14mmjhp
+/*/datum/design/ammolathe/a14mmjhp
 	name = "12.7mm JHP ammo box"
 	id = "a127mmjhp"
 	build_path = /obj/item/ammo_box/m14mm/jhp
 	materials = list(/datum/material/iron = 11000, /datum/material/blackpowder = 1500)
 	category = list("initial", "Basic Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:346]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 54]
```diff
@@ -436,12 +447,13 @@
...
 	category = list("initial", "Intermediate Ammo")
 
-/datum/design/ammolathe/c4570jhp
+/*/datum/design/ammolathe/c4570jhp
 	name = ".45-70 JHP ammo box"
 	id = "c4570jhp"
 	materials = list(/datum/material/iron = 12000, /datum/material/blackpowder = 2000)
 	build_path = /obj/item/ammo_box/c4570box/jhp
 	category = list("initial", "Intermediate Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:456]:
```suggestion
```

**Comment by Tron-117** [comment]
PLEASE I GET IT AAGHH

File: code/modules/research/designs/ammolathe_designs.dm [hunk 55]
```diff
@@ -457,19 +469,21 @@
...
 	category = list("initial", "Intermediate Ammo")
 
-/datum/design/ammolathe/a22hp
+/*/datum/design/ammolathe/a22hp
 	name = ".22lr HP ammo box"
 	id = "a22hp"
 	build_path = /obj/item/ammo_box/m22/hp
 	materials = list(/datum/material/iron = 8000, /datum/material/blackpowder = 2000)
 	category = list("initial", "Intermediate Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:478]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 56]
```diff
@@ -457,19 +469,21 @@
...
 
-/datum/design/ammolathe/a9mmap
+/*/datum/design/ammolathe/a9mmap
 	name = "9mm AP ammo box"
 	id = "a9mmap"
 	build_path = /obj/item/ammo_box/c9mm/ap
 	materials = list(/datum/material/iron = 15000, /datum/material/titanium = 3750, /datum/material/blackpowder = 2000)
 	category = list("initial", "Intermediate Ammo")
+*/
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:486]:
```suggestion
```

File: code/modules/research/designs/ammolathe_designs.dm [hunk 57]
```diff
@@ -478,7 +492,7 @@
 	materials = list(/datum/material/iron = 15000, /datum/material/blackpowder = 2000)
 	category = list("initial", "Intermediate Ammo")
 
-/datum/design/ammolathe/a357jfp
+/*/datum/design/ammolathe/a357jfp
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:495]: This. (Line 495 to 515)

File: code/modules/research/designs/ammolathe_designs.dm [hunk 58]
```diff
@@ -581,7 +596,7 @@
 	build_path = /obj/item/ammo_box/c9mm/op
 	category = list("initial", "Advanced Ammo")*/
 
-/datum/design/ammolathe/a4570swc
+/*/datum/design/ammolathe/a4570swc
```
Comment (Looreydev) [inline code/modules/research/designs/ammolathe_designs.dm:599]: This. (Line 599 to 619)

**Review by Looreydev — CHANGES_REQUESTED** [review]
Tl;dr - Remove the comments, delete the things entirely. The chance they will sit there and rot, and be messy is too high. Another point - we are using git, a version control system, which makes it super easy to to revert back to a point and just bring back changes / deleted files thanks to its history.

**Comment by Tron-117** [comment]
This PR is good to merge, but it'll merge with Adam's gun rework and my Armor Rework PR at the same time.

**Review by Looreydev — APPROVED** [review]

**Comment by AdamElTablawy** [comment]
this is mergeable assuming it's updated
